content/en/integrations/guide/source-code-integration.md

```diff
@@ -125,42 +113,31 @@ Reporting commit 007f7f466e035b052415134600ea899693e7bb34 from repository git@gi
 {{% /tab %}}
 {{< /tabs >}}
 
-## Links to Git
-
-#### Stack traces
-
-In [Error Tracking][1] and on APM error spans, you can directly access links to repositories from your stack traces.
-
-1. Navigate to **APM** > **Error Tracking**.
-2. Click on an issue. The **Issue Details** panel appears to the right.
-3. Under **Latest available errors**, hover over a frame. The **View** button appears to the right and directs you to GitHub.
-
-{{< img src="integrations/guide/github_apps/stacktrace-link-to-git.png" alt="Inline code snippet" style="width:90%;">}}
-
-##### Inline source code
-
-If you are a GitHub SaaS user, install Datadog's [GitHub integration][2] to directly inline code snippets from your GitHub repository in your stack traces.
+## Examples of links to Git repositories
 
-When specifying your permissions in the integration tile, enable Datadog read permissions to **Contents**.
+### Inline source code
```

Comment:
Call out Links to your repositories (whatever source code manager is) first and add steps 1 to 5. Callout Error Tracking and Continuous Profiler. links to git providers: error tracking, CP.

Then second section is Inline source code. error tracking with that snapshot, need the github integration for this to happen.